app/javascript/src/ActiveDocs/components/ActiveDocsSpec.jsx

```diff
@@ -0,0 +1,18 @@
+// @flow
+
+import React from 'react'
+
+import { createReactWrapper } from 'utilities/createReactWrapper'
+import SwaggerUI from 'swagger-ui-react'
+
+import 'swagger-ui-react/swagger-ui.css'
```

Comment:
If we could avoid injecting css in the HTML page that would be better
Especially we would like to use the CDN

Reply:
Actually, we are now aiming to stop forking and modifying third party assets and providing them through the CDN (which eventually will cease to exist) promoting having everything in our packange.json and extend it through plugins (in the case of swagger ui 3). That style bundle is provided bu the same swagger lib imported 2 lines above, so it's totally OK to provide it in our javascript component. Easy to maintain, test, tweak, track license, etc.